tests/queries/0_stateless/03448_sparse_grams_bloom_filter.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+# Tags: no-replicated-database
+# Tag no-replicated-database: grep -c
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+$CLICKHOUSE_CLIENT --query="DROP TABLE IF EXISTS bloom_filter_idx;"
```

Comment:
I think we can convert this to a regular .sql test.

Reply:
I think we shouldn't do this, because we are using `grep` in this test. If we replace this test with sql query, we will collect all statistics and they will contain non-constant values like 'elapsed'